lib/cartopy/_crs.pyx

```diff
@@ -336,14 +336,14 @@ cdef class CRS:
             result[:, 2] = z
 
         if src_crs.is_geodetic():
-            result = np.deg2rad(result)
+            result[:, :2] = np.deg2rad(result[:, :2])
```

Comment:
> That should limit some extraneous copying.

I guess it would save a memcopy/temporary, but I suspect it is in the noise. I'm happy if you want to pursue it, but personally am not overly concerned. Your call.